### ISSUE TITLE
[SideNavigation][A11y]: The state as ‘Checked/ Selected’ information is not read on accessing the items

### ISSUE THREAD

**Issue by AlekseiSch** [SAP/ui5-webcomponents#10917]
### Bug Description

**Observed Behavior:** With Voiceover enabled, the state as ‘Checked/ Selected’ information is not read on accessing the menu items.

https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/layouts-floorplans-sidenavigation--default

### Affected Component

SideNavigation

### Expected Behaviour

With Voiceover enabled, state of the UI should be read out to screen reader users.
Information as ‘Checked/selected’ should be read out when menu items are accessed.

### Priority

None

### UI5 Web Components Version

2.7.0

### Browser

Safari

### Operating System

iOS 18.1.1

### Additional Context

Device Name: Apple iPhone 13
Operating System: iOS 18.1.1
Screen Reader: Voiceover 18.1.1
Browser: Safari

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-rd ,

could you please check this issue?
The issue can be reproduced using the provided link.

Thanks,
Todor

**Comment by GerganaKremenska** [comment]
Hello @AlekseiSch, 

I retested and for the selected item the "selected" state is anounced.
@Todor-ads, how did you test and reproduced the issue?

![Image](https://github.com/user-attachments/assets/a2f5ce95-043d-46a5-97c0-f0f06d391bd9)

@AlekseiSch If you are able to repsroduce the issue, please add deatils steps on how to reproduce and reopen the issue.
I am closing it since currently there is no action on development side.

Best Regards,
Gergana

**Comment by AlekseiSch** [comment]
> Hello [@AlekseiSch](https://github.com/AlekseiSch),
> 
> I retested and for the selected item the "selected" state is anounced. [@Todor-ads](https://github.com/Todor-ads), how did you test and reproduced the issue?
> 
> ![Image](https://github.com/user-attachments/assets/a2f5ce95-043d-46a5-97c0-f0f06d391bd9)
> 
> [@AlekseiSch](https://github.com/AlekseiSch) If you are able to repsroduce the issue, please add deatils steps on how to reproduce and reopen the issue. I am closing it since currently there is no action on development side.
> 
> Best Regards, Gergana

Hello @GerganaKremenska. For now, it's reproducible in Chrome, NVDA. State is announced as current page, without selected

![Image](https://github.com/user-attachments/assets/b6e874e2-94a2-44af-8ac0-bde94aef4918)

**Comment by dreamt12** [comment]
@GerganaKremenska Please reopen the ticket based on @AlekseiSch comment above. Both of us don't have permissions to reopen. It would be best to wait with closing it until we have a chance to provide feedback.

**Comment by GerganaKremenska** [comment]
Hello @AlekseiSch  and @dreamt12, 

The reported issue is about Voice Over after testing and concluding that there is no issue the ticket shoould be closed.

The situation now is that there is a change of the used screen reader and the initialy reported issue is no longer the same, which should be opened in a new issue.

ui5-side navigation is folowing aria best pracites and implement by these [recomentadiotns](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-navigation/). Difrent venodrs(screen readers), have difrrent implemtnations and we can not influnce if the vendor has an issue or this is the correct behavior accroding to the vendor. 

Currently ui5-side-navigation and the exmaple in the best practice app have the same accessibility and NVDA does not read "selected". 
There is already a reported [issue](https://github.com/nvaccess/nvda/issues/14352) to NVDA regarding this.

If you have othre issues please reopened them as new ones. 

The one about voice over is not reproducable,  and the NVDA issue is reported to the screen reader.

Closing the issue as there is no action on development side.

Best Regards,
Gergana
